src/components/Premium/PremiumBrand.tsx

```diff
@@ -1,32 +1,104 @@
+import { useEffect, useState } from 'react';
 import styled from 'styled-components';
 
-import { BrandNameListIcon } from '../../assets/icon';
-import { 꼬달리Img, 다비네스Img, 달팜Img, 더바디샵Img } from '../../assets/image';
+import { BrandSortIcon, BrandSortReverseIcon } from '../../assets/icon';
+import { BrandData } from '../../types/brand';
+import { getBrandData } from '../../utils/lib/brand';
+
+import PremiumBrandCard from './PremiumBrandCard';
 
 const PremiumBrand = () => {
+  
+  // 1. API 연결하여 브랜드 리스트 받아오기
+  
+  const [PremiumBrandList, setBrandList] = useState<BrandData[]>([]); 
+
+  const getBrandList = async () => {
+    try {
+      const { data: { data } } = await getBrandData();
+      setBrandList(data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    getBrandList();
+  }, []);
+  
+  // 2. 정렬 버튼 누르면 정렬
+
+  const [isAscending, setIsAscending] = useState<boolean>(true);
+  
+  const SortBrandList = () => {
+    const sortedList = [...PremiumBrandList].sort((a, b) => {
```

Comment:
```suggestion
    const sortedList = PremiumBrandList.sort((a, b) => {
```
요긴 이렇게 그냥 List 바로 넣어줘도 괜찮지 않오?!